### ISSUE TITLE
Improve MLflow CLV support

### ISSUE THREAD

**Issue by wd60622** [pymc-labs/pymc-marketing#1259]
Would like to include some similar CLV wrapper for save and load. Maybe some auto logging items as well. 

Anything you have in mind? I dont mind implementing. @ColtAllen

**Comment by ColtAllen** [comment]
Is the model `idata` object being logged? May want to add an option on including customer data for that first, otherwise the size of files being logged could be very large.

**Comment by ColtAllen** [comment]
> optionally saving only the non-customer data? Is that a functionality that already exists with the `save` method?

Yes, right now this functionality is unsupported. There will be some plotting functionality lost if loading a model without RFM customer data, but these plots can be ran & logged to MLFlow before saving.

**Comment by wd60622** [comment]
Makes sense. I can see which walls we hit and try to replicate the save and load that will come with MMM. If the first iteration is saving the whole training data, so be it. Do you have an issue for excluding the training data from `save` method?
